infra/base-images/base-builder/indexer/clang_wrapper.py

```diff
@@ -176,18 +176,26 @@ def remove_flag_and_value(argv: list[str],
   return None
 
 
+def is_suffix_match(needle: str, haystack: str):
+  haystack_path, haystack_suffix = haystack.split(':', 1)
+  needle_path, needle_suffix = needle.split(':', 1)
+  haystack_path = os.path.abspath(haystack_path)
```

Comment:
are we sure if the cwds are the same in the context that wrote these paths? otherwise abspath might be wrong right?

Reply:
i might split out the .so fixes into another PR and get that merged first.